Parsers/ASimWebSession/Parsers/ASimWebSessionCiscoMeraki.yaml

```diff
@@ -0,0 +1,228 @@
+      | extend
+          Epoch = tostring(Parser[0]),
+          Device = tostring(Parser[1])
+      | extend
+          EpochTimestamp = split(Epoch, ".")
+      | extend
+          EventStartTime = unixtime_seconds_todatetime(tolong(EpochTimestamp[0]))
+      | extend
+          disposition= iff(
+                  disposition has '"',
+                  replace_string(tostring(disposition), '"', ''),
+                  replace_string(tostring(disposition), "'", "")
+              ),
+          action= iff(
+              action has '"',
+              replace_string(tostring(action), '"', ''),
+              replace_string(tostring(action), "'", "")
+          ),
+          agent= iff(
+            agent has '"',
+            replace_string(tostring(agent), '"', ''),
+            replace_string(tostring(agent), "'", "")
+        ),
+          sha256= iff(
+              sha256 has '"',
+              replace_string(tostring(sha256), '"', ''),
+              replace_string(tostring(sha256), "'", "")
+          ),
+          fsha256= iff(
+              sha256 has '"',
+              replace_string(tostring(fsha256), '"', ''),
+              replace_string(tostring(fsha256), "'", "")
+          ),
+          name= iff(
+            name has '"',
+            replace_string(tostring(name), '"', ''),
+            replace_string(tostring(name), "'", "")
+        ),
+          mac= iff(
+          mac has '"',
+          replace_string(tostring(mac), '"', ''),
+          replace_string(tostring(mac), "'", "")
+      ),
+          url= iff(
+          url has '"',
+          replace_string(tostring(url), '"', ''),
+          replace_string(tostring(url), "'", "")
+      ),
+          urls= iff(
+            urls has '"',
+            replace_string(tostring(urls), '"', ''),
+            replace_string(tostring(urls), "'", "")
+        )
+      | lookup ActionLookup on action
+      | extend Url = coalesce(url, urls)
+      | extend
+          EventResult=case(
+                  action in~("allow", "log", "accept") or LogType == "urls",
+                  "Success",
+                  action in~("block", "deny", "quarantine"),
+                  "Failure",
+                  ""
+              ),
+          EventSeverity=case(
+                    action in~("block", "deny", "quarantine") and disposition == "malicious",
+                    "Medium",
+                    action in~("allow", "log", "accept") and disposition == "malicious",
+                    "High",
+                    action in~("block", "deny", "quarantine") and disposition != "malicious",
+                    "Low",
+                    "Informational"
+                ),
+          Ip=extract("(([0-9]{1,3})\\.([0-9]{1,3})\\.([0-9]{1,3})\\.(([0-9]{1,3})))", 1, url)
+      | extend SrcIpAddr = iff(
+                          src has ".",
+                          split(src, ":")[0], 
+                          coalesce(temp_srcip, src)
```

Comment:
What's the need for temp_srcip value? we are extracting this value from 'src' itself using parse at line 65. Can we directly use 'src' ?